src/workerd/api/actor-state.c++

```diff
@@ -723,6 +725,99 @@ jsg::Promise<jsg::Value> DurableObjectState::blockConcurrencyWhile(jsg::Lock& js
   return IoContext::current().blockConcurrencyWhile(js, kj::mv(callback));
 }
 
+void DurableObjectState::acceptWebSocket(
+    jsg::Ref<WebSocket> ws,
+    jsg::Optional<kj::Array<kj::String>> tags) {
+  JSG_ASSERT(!ws->isAccepted(), Error,
+      "Cannot call `acceptWebSocket()` if the WebSocket was already accepted via `accept()`");
+  JSG_ASSERT(ws->pairIsAwaitingCoupling(), Error,
+      "Cannot call `acceptWebSocket()` on this websocket because its pair has already been "\
```

Comment:
Nit... inconsistent casing in `WebSocket` in previous error and `websocket` in this one.

Reply:
Marking resolved since this was fixed in the hibernation manager PR